File: src/main/java/org/c3lang/intellij/findUsages/C3FindUsagesHandlerFactory.java
package org.c3lang.intellij.findUsages;

import com.intellij.find.findUsages.FindUsagesHandler;
import com.intellij.find.findUsages.FindUsagesHandlerFactory;
import com.intellij.psi.PsiElement;
import kotlin.NotImplementedError;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

// TODO
public final class C3FindUsagesHandlerFactory extends FindUsagesHandlerFactory
{
	@Override
	public boolean canFindUsages(@NotNull PsiElement element)
	{
		throw new NotImplementedError("An operation is not implemented: Not yet implemented");
	}

	@Override
	public @Nullable FindUsagesHandler createFindUsagesHandler(
		@NotNull PsiElement element,
		boolean forHighlightUsages)
	{
		throw new NotImplementedError("An operation is not implemented: Not yet implemented");
	}
}

File: src/main/java/org/c3lang/intellij/findUsages/C3FindUsagesHandlerFactory.kt
package org.c3lang.intellij.findUsages

import com.intellij.find.findUsages.FindUsagesHandler
import com.intellij.find.findUsages.FindUsagesHandlerFactory
import com.intellij.psi.PsiElement

// TODO
class C3FindUsagesHandlerFactory : FindUsagesHandlerFactory() {

    override fun canFindUsages(element: PsiElement): Boolean {
        TODO("Not yet implemented")
    }

    override fun createFindUsagesHandler(element: PsiElement, forHighlightUsages: Boolean): FindUsagesHandler? {
        TODO("Not yet implemented")
    }

}